_episodes/09-pull-requests.md

```diff
@@ -84,32 +128,66 @@ $ git remote -v
 ~~~
 origin	https://github.com/USERNAME/countries.git (fetch)
 origin	https://github.com/USERNAME/countries.git (push)
-upstream	https://github.com/McMahonLab/countries.git (fetch)
-upstream	https://github.com/McMahonLab/countries.git (push)
+upstream	https://github.com/INSTRUCTOR-GIVEN/countries.git (fetch)
+upstream	https://github.com/INSTRUCTOR-GIVEN/countries.git (push)
 ~~~
 {: .output}
 
-Now that we have this setup done, which is only needed the first time, we can now suggest changes to this repo.
-Each person will add info about a specific country to the repo.
-Since you cloned the repo a template file with info on the United States has been added.
-First we will update our local version of the repo, with this change.
+Now that we have this setup done we will be able to suggest 
+changes to this repo using a pull request.
+Each person will add a new file with info about a new country in it.
+
+The instructor will now add a single file to the repository containing 
+information about the the United States.
+
+Next, we will update our local version of the repo, with this change by `pull`ing down
+the changes from the 'upstream' version's master branch.
 
 ~~~
 $ git pull upstream master
 ~~~
 {: .bash}
 
-Now your local version of the repo is updated but our forked version of the repo is not yet up to date.
-Now we will update the our forked version.
+Now your local version of the repo is updated but our forked version of the 
+repo is not yet up to date.
+You can reload your fork in github and see it does not contain the new 
+`united_states.txt` file.
+Now we need to update the our forked version.
+To do so we can `push` the changes in our local version to the master branch of our fork, 
+called 'origin'.
 
 ~~~
 $ git push origin master
 ~~~
 {: .bash}
 
-Now you can add your country to repo.
-First copy `united_states.txt` and change the name to the name of your chosen country.
-Then edit the contents to reflect the info of your country.
+Now let's each add a new country to the repository.
+First let's make a new branch to work on.  This will keep our 'master' version
+in sync with the authoritative version of the repository.
+We can name our branch descriptively after the country we will be adding 
+Mine will be `addFrance` since I'll be working with France.
+Please pick a different country and shout it out (or add it to the etherpad) 
+so no one else chooses the same one.
+We will create the branch and switch into in one step 
+as we learned earlier in the branching lesson.
+Finally before we proceed to adding the new file, we will double 
```

Comment:
I'm feeling that this paragraph should be broken up and the commands separated for clarity and reduced cognitive load here. I'm thinking:

  - checkout in one step description. end paragraph
  - `git checkout -b addFrance`
  - "Finally before we proceed..."
  - `git branch`
  - output from `git branch` command